lib/default.nix

```
@@ -0,0 +1,79 @@
+{lib, ...}: let
+in {
+  genRegistry = pkgs: let
+    registerPackage = name: value: let
+      safeValue = builtins.tryEval value;
+
+      safeRegistryValue = builtins.tryEval (builtins.deepSeq registryValue registryValue);
+      registryValue = {
+        pname = value.pname or value.name or null;
```

```suggestion
        pname = value.pname or (builtins.parserDrvName value.name).name or null;
```

otherwise we might get versions in there.

---

fixed in https://github.com/replit/rippkgs/commit/95c327427e6d26741749d60115bc2fee8969db87